.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - master
```

Comment:
@gutakk Isn't the branch name `main`? 🤔

Reply:
**** myself, that was sloppy. Excellent catch. Fixed in 1a9036fd6c1f2c693d083f6503df8bcf32111e34